Source/JavaScriptCore/wasm/WasmBBQJIT.cpp

```diff
@@ -5733,15 +5733,14 @@ class BBQJIT {
         Jump tierUp = m_jit.branchAdd32(CCallHelpers::PositiveOrZero, TrustedImm32(TierUpCount::functionEntryIncrement()), Address(m_scratchGPR));
         MacroAssembler::Label tierUpResume = m_jit.label();
         auto functionIndex = m_functionIndex;
-        addLatePath([tierUp, tierUpResume, functionIndex](BBQJIT&, CCallHelpers& jit) {
+        addLatePath([tierUp, tierUpResume, functionIndex, this](BBQJIT&, CCallHelpers& jit) {
```

Comment:
Instead of capturing `this`, use `BBQJIT&`'s parameter.

Reply:
This is probably best. Added it in the latest push.